docs/worker.md

```diff
@@ -80,7 +80,7 @@ do {
 
     // Call the garbage collector to reduce the chances of it being triggered in the middle of a page generation
     gc_collect_cycles();
-} while ($running && !(isset($_SERVER['MAX_REQUESTS']) && ++$nbRequests >= $_SERVER['MAX_REQUESTS']));
+} while ($running && isset($_SERVER['MAX_REQUESTS']) && (++$nbRequests <= $_SERVER['MAX_REQUESTS']));
```

Comment:
It might be worth considering a for-loop instead of a while loop:

```php
for($nbRequests = 0, $running = true; $nbRequests < $_SERVER['MAX_REQUESTS'] && $running; $nbRequests++) {
 // do request stuff
}
```

as well as moving the `$handler` out of the loop so that a new closure isn't created on every request.

Reply:
Maybe with pre-increment because it's sligthly faster. when converted to OPCODES @withinboredom